package.json

```diff
@@ -104,22 +105,24 @@
     "react-fontawesome": "^1.1.0",
     "react-helmet": "^3.1.0",
     "react-hotkeys": "^0.8.0",
-    "react-look": "^1.0.0-beta9",
+    "react-look": "file:../react-look-1.0.0-beta9.tgz",
```

Comment:
Should we check in our binary until then? Perhaps in something like `./mod_bin/react-look...tgz`?